examples/tutorial.dx

```diff
@@ -292,9 +292,9 @@ def tableAdd' (x : m => n => Float32) (y : m => n => Float32) : m => n => Float3
 
 ' To run this section, move the following binary files to examples:
 
-' `wget https://github.com/srush/learns-dex/raw/main/mnist.bin`
+' `wget http://yann.lecun.com/exdb/mnist/t10k-images-idx3-ubyte.gz; gunzip t10k-images-idx3-ubyte.gz`
```

Comment:
Isn't `unzip` a bit more standard for this? I think it's equivalent to gunzip on most Linuxes, but also works on BSD-based systems (macOS).

Reply:
Hmm, is that true? `unzip` doesn't seem to work for me linux for .gz files. Also `gunzip` seems to install fine in the macOS CI test. But I don't have a mac to test on. Does unzip work there?

Reply:
I confirm the following works on macOS:
```console
$ wget http://yann.lecun.com/exdb/mnist/t10k-images-idx3-ubyte.gz; gunzip t10k-images-idx3-ubyte.gz
# Produces unzipped t10k-images-idx3-ubyte, 7.5M.
```
Hope this helps!